schemas/bridging/raiseIntentResultResponse.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/raiseIntentResultResponse.schema.json",
+  "title": "RaiseIntentResultResponse",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeResponse.schema.json"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "raiseIntentResultResponse"
+        },
+        "payload": {
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "context": {
+                  "$ref": "../context/context.schema.json"
+                }
+              },
+              "required": ["context"],
+              "additionalProperties": false
+            },
+            {
+              "type": "object",
+              "properties": {
+                "channel": {
+                  "$ref": "../api/channel.schema.json"
+                }
+              },
+              "required": ["channel"],
+              "additionalProperties": false
+            }
+          ]
+        },
```

Comment:
add intent result layer to response. Both channel and context can be omitted - I think that's clearer with an empty object in the oneOf, but if it doesn't work drop it and the required elements on context and channel and that will achieve the same thing


```suggestion
        "payload": {
          "type": "object",
          "properties": {
            "intentResult": {
              "oneOf": [
                {
                  "type": "object",
                  "properties": {
                    "context": {
                      "$ref": "../context/context.schema.json"
                    }
                  },
                  "required": ["context"],
                  "additionalProperties": false
                },
                {
                  "type": "object",
                  "properties": {
                    "channel": {
                      "$ref": "../api/channel.schema.json"
                    }
                  },
                  "required": ["channel"],
                  "additionalProperties": false
                },
                {
                  "type": "object",
                  "properties": { }
                  },
                  "additionalProperties": false
                }
              ]
            }
          },
          "required": ["intentResult"],
          "additionalProperties": false
        },
```